HinglishDistilBERTMultilingual.ipynb

```diff
@@ -42,7 +42,7 @@
         "!pip install transformers\n",
         "!git clone https://github.com/NirantK/Hinglish.git\n",
         "%cd Hinglish\n",
-        "!git checkout all_berts"
+        "!git checkout transformers"
```

Comment:
Should this branch change be in the `master` branch?

Reply:
Resolved by 4265454